### PR TITLE
Update committee.py

### PR DESCRIPTION
**Pull request by ojimba01**
Deleted expand_zip function, edited process_commitee function to be updated with the CSV headers, and edited create_division function to adjust to the new ward_div format. Additionally updated the requirements.txt so the package dependencies are up to date.

### CODE REVIEW

File: data-scripts/committee.py [hunk 1]
```diff
@@ -1,32 +1,28 @@
...
-
 def create_ward(row):
-    # Get characters before -, trim leading zeros, convert to int
     return int(row['ward_div'].split('-')[0].lstrip('0'))
 
 def create_division(row):
+    # Delete "DEM" or "REP" from the end of the name
     # Get characters after -, trim leading zeros, convert to int
-    return int(row['ward_div'].split('-')[1].lstrip('0'))
+    return int(row['ward_div'].split('-')[1].lstrip('0').strip('DEM'))
```
Comment (pwolanin) [inline data-scripts/committee.py:9]: need to handle both parties?

File: data-scripts/requirements.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
-click==6.7
-petl==1.1.1
-ratelimit==1.4.1
-contentful_management==1.4.0
-tqdm==4.19.2
+click>=6.7,<=8.1.3
+petl==1.1.1,<=1.7.10
```
Comment (pwolanin) [inline data-scripts/requirements.txt:2]: if this has == does the max matter?

File: data-scripts/committee.py (hunk 1 above)
Comment (ojimba01) [reply data-scripts/committee.py:9]: Just fixed this!

File: data-scripts/requirements.txt (hunk 2 above)
Comment (ojimba01) [reply data-scripts/requirements.txt:2]: That's my bad! as long as it's equal to or above the requirements then the scripts work!

**Comment by ojimba01** [comment]
I just fixed these errors! I'll make a new pull request!

**Pull request closed without merging**